packages/@aws-cdk/app-staging-synthesizer-alpha/lib/default-staging-stack.ts

```diff
@@ -17,11 +18,24 @@ import {
   StackProps,
 } from 'aws-cdk-lib/core';
 import { StringSpecializer } from 'aws-cdk-lib/core/lib/helpers-internal';
+import * as cxapi from 'aws-cdk-lib/cx-api';
+import { Construct } from 'constructs';
 import { BootstrapRole } from './bootstrap-roles';
 import { FileStagingLocation, IStagingResources, IStagingResourcesFactory, ImageStagingLocation } from './staging-stack';
 
 export const DEPLOY_TIME_PREFIX = 'deploy-time/';
 
+/**
+ * This is a dummy construct meant to signify that a stack is utilizing
+ * the DefaultStagingStack. It does not do anything, and is not meant
+ * to be created on its own.
+ */
+export class UsingDefaultStagingStack extends CfnWaitConditionHandle {
```

Comment:
I'm ready for you to tell me what the right construct to extend is. AFAIK, we need to extend an actual CFN resource, and `WaitConditionHandle` looks like the simplest. Not sure how users react when this is added though.

Reply:
> AFAIK, we need to extend an actual CFN resource

We do not. An empty class that extends `Construct` is fine.

Reply:
I think I'd like this to represent

```suggestion
export class UsingAppStagingSynthesizer extends CfnWaitConditionHandle {
```